techniques/general/Gxxx-highly-visible-focus-indicator.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Using an author-supplied, highly visible focus indicator</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></link></head><body><h1>Using an author-supplied, highly visible focus indicator</h1><section class="meta"><p class="id">ID: G195</p><p class="technology">Technology: general</p><p class="type">Type: Technique</p></section><section id="applicability"><h2>When to Use</h2>
+    <p>Generally applicable.</p>
+ </section><section id="description"><h2>Description</h2>
+    <p>The objective of this technique is enhance the focus indicator in the browser, by creating a highly visible one in the content. This technique is very similar to <a href="G195">G195</a>, but with a stronger indicator to meet <a href="focus-appearance-enhanced">Focus Appearance (Enhanced)</a>.</p>
+ </section>
+ <section id="examples">
+    <h2>Examples</h2>
+    <section class="example">
+       <h3>Links</h3>
+          <p>A Web page has a dark background color and light text and links. When focus lands on a link, the link is outlined with a bright yellow line, 3 pixels wide.</p>
+    </section>
+    <section class="example">
+       <h3>Form Elements</h3>
+       
+          <p>A Web page includes a form inside a table. The borders of both the table and the form elements are thin, black lines. When focus lands on a form element, the element is outlined with a 5 pixel red line that is partially transparent. The red is equivalent to a hex color of #CA0000, providing a 6:1 contrast ratio with the white background.</p>
+       
+    </section>
+    <section class="example">
+       <h3>Menus</h3>
+       
+          <p>A Web page includes an interactive menu with sub-menus. A user can move focus in the menu using the arrow keys. As focus moves, the currently focused menu item changes its background to a different color, which has a 15:1 contrast ratio with the surrounding items (whilst maintaining text contrast).</p>
+       
+    </section>
+ </section><section id="tests"><h2>Tests</h2>
+    <section class="procedure"><h3>Procedure</h3>
+       <p>For each user interface element on the page that should receive keyboard focus:</p>
```

Comment:
same suggested change here and in the next `<li>` about `component` rather than `element`